doc/_sphinx/extensions/flutter_app.py

```diff
@@ -178,13 +178,19 @@ def _ensure_compiled(self):
 
     def _compile_source(self):
         try:
-            subprocess.run(
+            """ subprocess.run(
                 ['flutter', 'build', 'web'],
                 stdout=subprocess.PIPE,
                 stderr=subprocess.STDOUT,
                 cwd=self.source_dir,
                 check=True,
-            )
+            ) """
+            cwd = os.getcwd()
+            # Print the current working directory
+            print("Current working directory: {0}".format(cwd))
+            abspath = os.path.abspath("../flame/examples")
+            os.chdir(abspath)
+            subprocess.run("flutter build web", shell=True)
```

Comment:
Try
```python
            subprocess.run(
                "flutter build web",
                shell=True,
                stdout=subprocess.PIPE,
                stderr=subprocess.STDOUT,
                cwd=self.source_dir,
                check=True,
            )
```

Reply:
This worked as well.  Do you think we still need to change directory?  I commented it out and it still ran, but I had issues earlier about not finding the `pubspec.yaml` file for flutter build. 

This is what I currently have and it seems to be working ok:

```python
def _compile_source(self):
        try:
            subprocess.run(
                "flutter build web",
                shell=True,
                stdout=subprocess.PIPE,
                stderr=subprocess.STDOUT,
                cwd=self.source_dir,
                check=True,
            )
        except subprocess.CalledProcessError as e:
            cmd = ' '.join(e.cmd)
            raise self.error(
                f'Command `{cmd}` returned with exit status {e.returncode}\n' +
                e.output.decode('utf-8'),
            )
```

Reply:
The `cwd` argument means "current working directory", so it should take care of that.

Reply:
Ah, I didn't even read the arguments. Good to know.